### PR TITLE
fix: updated instructions for install-and-set-up-mongoose

### LINKED ISSUE #37516: Install and Set Up Mongoose - the right way to connect Mongoose
Lesson - https://freecodecamp.org/learn/apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose

In this lesson you should Install and Set Up Mongoose. Setting up Mongoose in the .env file you should keep in mind, that url to your MongoDB must be saved as STRING. Check it in [docs](https://mongoosejs.com/docs/connections.html).
If you save it as:
`MONGO_URI=mongodb+srv://admin:6813378dima@cluster0-2qeus.gcp.mongodb.net/test?retryWrites=true&w=majority`
it will cause fail tests!
Screenshot
The right way is:
`MONGO_URI='mongodb+srv://admin:6813378dima@cluster0-2qeus.gcp.mongodb.net/test?retryWrites=true&w=majority'`

Please make corrections in [getting hint](https://www.freecodecamp.org/forum/t/freecodecamp-challenge-guide-install-and-set-up-mongoose/301540) and pinpoint this in the lesson.

### PR DESCRIPTION
**Pull request by classicmatsuo**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #37516

### CODE REVIEW

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ forumTopicId: 301540
 
 ## Description
 <section id='description'>
-Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Connect to the database using the following syntax:
+Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI ( must be saved as a STRING) in the private <code>.env</code> file as MONGO_URI. Connect to the database using the following syntax:
```
Comment (RandellDawson) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md]:
```suggestion
Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single of double quotes and there should be no space between the variable and the `=`. Connect to the database using the following syntax:
```

Comment (classicmatsuo) [reply curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md]: typo 'of'. 'single or double'.

File: curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ forumTopicId: 301540
 
 ## Description
 <section id='description'>
-Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Connect to the database using the following syntax:
+Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single or double quotes and there should be no space between the variable and the `=` and the value. Connect to the database using the following syntax:
```
Comment (RandellDawson) [inline curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/install-and-set-up-mongoose.english.md]: Thanks for catching my typo on the other suggestion.  After reading this again, I think we need one more change.  You can commit this if you agree.

```suggestion
Add mongodb and mongoose to the project’s package.json. Then require mongoose. Store your MongoDB Atlas database URI in the private <code>.env</code> file as MONGO_URI. Surround the the URI with single or double quotes and make sure no space exists between both the variable and the `=` and the value and `=`. Connect to the database using the following syntax:
```

**Review by RandellDawson — APPROVED** [review]
Alright, I think this PR looks good now.

**Pull request merged**